ios/Classes/bindings.h

```diff
@@ -41,38 +41,58 @@ typedef struct wire_BlockchainConfig {
   union BlockchainConfigKind *kind;
 } wire_BlockchainConfig;
 
+typedef struct wire_ScriptAmount {
+  struct wire_uint_8_list *script;
+  uint64_t amount;
+} wire_ScriptAmount;
+
+typedef struct wire_list_script_amount {
+  struct wire_ScriptAmount *ptr;
+  int32_t len;
+} wire_list_script_amount;
+
+typedef struct wire_OutPoint {
+  struct wire_uint_8_list *txid;
+  uint32_t vout;
+} wire_OutPoint;
+
+typedef struct wire_list_out_point {
+  struct wire_OutPoint *ptr;
+  int32_t len;
+} wire_list_out_point;
+
 typedef struct DatabaseConfig_MEMORY {
 
 } DatabaseConfig_MEMORY;
 
-typedef struct wire_SqliteConfiguration {
+typedef struct wire_SqliteDbConfiguration {
   struct wire_uint_8_list *path;
-} wire_SqliteConfiguration;
+} wire_SqliteDbConfiguration;
 
 typedef struct DatabaseConfig_SQLITE {
-  struct wire_SqliteConfiguration *config;
```

Comment:
Swift is not my specialty, but I'm wondering if you can tell me what the `wire` in the name at the beginning of all the arguments does?

Reply:
That's autogenerated by flutter rust bridge